### PR TITLE
Removed MAX_STEPS from sim

### PR DESCRIPTION
**Pull request by asgibson**
Limiting the steps was making demos not work correctly Removed constant  
Unit tests updated to reflect change  
  removed hardcoded values from tests for steps  
  removed hardcoded values from tests for diagnosis interval  
    tested and used DIAGNOSIS_INTERVAL constant in tests

### CODE REVIEW

File: onair/src/run_scripts/sim.py [hunk 1]
```diff
@@ -47,6 +46,7 @@ def run_sim(self, IO_Flag=False, dev_flag=False, viz_flag = True):
             if self.agent.mission_status == 'RED':
                 if last_fault == time_step - 1: #if they are consecutive
                     if (time_step - last_diagnosis) % DIAGNOSIS_INTERVAL == 0:
+                        print(time_step)
```
Comment (the-other-james) [inline onair/src/run_scripts/sim.py:49]: Did you mean to keep this print?

File: test/onair/src/run_scripts/test_sim.py [hunk 2]
```diff
@@ -123,19 +127,21 @@ def test_Simulator_run_sim_prints_wait_message_when_given_IO_Flag_is_the_str_str
...
     # Arrange
     cut = Simulator.__new__(Simulator)
     cut.simData = MagicMock()
     cut.agent = MagicMock()
 
+    num_fake_steps = pytest.gen.randint(1, 100) # from 1 to 100 arbitrary for fast test
     fake_diagnosis = MagicMock()
     fake_next = MagicMock()
     fake_IO_Flag = MagicMock()
+    side_effects_for_has_more = [True] * (num_fake_steps) + [False]
```
Comment (the-other-james) [inline test/onair/src/run_scripts/test_sim.py:140]: That's cool,

**Review by the-other-james — CHANGES_REQUESTED** [review]
Looks good! Just not sure that one print should be there.

Tests pass on my machine.

File: onair/src/run_scripts/sim.py (hunk 1 above)
Comment (asgibson) [reply onair/src/run_scripts/sim.py]: nope, fixed

**Review by the-other-james — APPROVED** [review]

**Pull request merged**